components/database.js

```diff
@@ -25,6 +26,25 @@ export default class Database {
     }
   }
 
+  static async getUser(uname, pwd) {
```

Comment:
instead of parsing to JSON you can just iterate the object like in
https://github.com/msd-d/PetHub/blob/57595c8502dc6f188a95f7cdd918a574be7d7665/components/views/post.js#L66

Reply:
Huh, neat!

Reply:
Jeg tror ikke det ville virke i det her tilfælde. Ved at gøre som i `dataCheck` kan man ikke koble et `username` til et specifikt `password`, så man kan ikke være sikker på at det `password` man checker imod rent faktisk hører til det givne `username`.

Reply:
I see, nevermind then 🐱